### PR TITLE
Addition of Higher Order Component to verify authentication

### LINKED ISSUE #155: "File upload" heading is missing on file management page
1. Login to the CDN device with valid credential 
2. Click on file management 
3. File upload heading is missing on file management page 

Expected Result: File upload heading should display on file management page 

Actual Result: File upload heading is not displaying on file management page

### LINKED ISSUE #182: Refresh page the admin portal page logs out
1. Connected to openRAP network
2. Opened the admin portal page 
3. Clicked on refresh page it logs out from the admin portal page

### PR DESCRIPTION
**Pull request by SumukhaKV**
Fixes #155, #182, #244, #268, #275 and #276

### CODE REVIEW

File: devmgmtui/src/components/VerifyAuthentication.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+export default function(ComposedComponent) {
+    class Authentication extends Component {
+        static contextTypes = {
```
Comment (rishabh1403) [inline devmgmtui/src/components/VerifyAuthentication.js]: what is this used for ?

Comment (SumukhaKV) [reply devmgmtui/src/components/VerifyAuthentication.js]: It is a class level property which provides any piece of code in the application the ability to reference _Authentication.contextTypes_.
The HOC still works even if we eliminate this piece of code and go ahead with _this.props.history.push('/')_

Comment (rishabh1403) [reply devmgmtui/src/components/VerifyAuthentication.js]: remove it if it's not required, I do not see any use of it.

Comment (SumukhaKV) [reply devmgmtui/src/components/VerifyAuthentication.js]: Alrighty.

**Pull request merged**